### PR TITLE
[page-headers] Page headers

### PR DESCRIPTION
**Pull request by mirisuzanne**
Baby-Giraffe

## [Trello card](https://trello.com/c/FddSuQNK/11-anything-we-want-to-change-about-the-page-headers-and-hero-images#)
- hero image
- page title
- page meta (subtitle, dates, byline, tags…)

I also did some small cleanup on the footer while I was in there…

## [PR Preview](https://deploy-preview-13--oddleventy.netlify.com/)
The home page doesn't have much of a page header, so go to `resources` and click through on a range of older and newer blog posts…

### CODE REVIEW

**Comment by mirisuzanne** [comment]
@SondraE There are still subtitles - we just ported the oldest articles first, and most of them didn't have subtitles. Check out the about page, or one of the newer articles like "CSS Custom Properties In The Cascade".  I made the line-height change on page titles, see what you think. We can't really go any tighter without the clipping issue Stacy mentioned on the other branch.

File: content/_includes/site/og.njk [hunk 1]
```diff
@@ -6,14 +6,14 @@
 <meta property="fb:app_id" content="{{ config.facebook_id }}" />
 <meta property="og:locale" content="{{ open_graph.locale }}" />
 <meta property="og:type" content="{{ og.type or open_graph.type }}" />
-<meta property="og:title" content="{{ og.title or meta_title }}" />
+<meta property="og:title" content="{{ og.title or meta.title }}" />
 <meta property="og:image" content="{{ og_image }}" />
```
[the remark this answers is not in the document]
Comment (mirisuzanne) [reply content/_includes/site/og.njk]: The link works if you append it to the proper preview domain - and the same approach has worked on my other 11ty sites. I'll fix the unnecessary `//` tho.

**Comment by mirisuzanne** [comment]
@stacyk I did a fair amount of cleanup if you want to check it again

**Comment by mirisuzanne** [comment]
[Anything we want to change about the page headers and hero images?](https://trello.com/c/FddSuQNK/11-anything-we-want-to-change-about-the-page-headers-and-hero-images)

File: content/_includes/page/header.njk [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{% import 'utility.macros.njk' as utility %}
+
+<header data-layout="header">
+  {% if image.src %}
+    {% set style = ['--hero-position', image.position] | join(':') if image.position else none %}
```
Comment (stacyk) [inline content/_includes/page/header.njk]: super smart!

**Comment by stacyk** [comment]
Looks good to me @mirisuzanne!

**Pull request merged**